lib/utils/messages.js

```diff
@@ -13,56 +13,60 @@ const redify = str => `${RED}${str}${RESET}`;
-  case 'SAVED_TO_JSON':
-    return `Data was saved into file ${args[0]}`;
-  case 'NO_METRICS':
-    return 'No expectation metrics were found';
-  case 'NO_EXPECTATION_ERROR':
-    return `Metric ${args[0]} has to have warn and error values`;
-  case 'NO_EXPECTATIONS_FOUND':
-    return `expectation flag set but no expectations found on config file`;
-  case 'NO_SHEET_TYPE':
-    return `Sheet type ${args[0]} is not available.`;
-  case 'NO_GOOGLE_SHEET_OPTIONS':
-    return `Some of options for submitting data to Google Sheets are absent`;
-  case 'NO_MESSAGE_PREFIX_TYPE':
-    return `No matching message prefix: ${args[0]}`;
-  case 'METRIC_IS_UNAVAILABLE':
-    return `Sorry, ${args[0]} metric is unavailable`;
-  case 'ttfcp':
-    return 'First Contentful Paint';
-  case 'ttfmp':
-    return 'First Meaningful Paint';
-  case 'psi':
-    return 'Perceptual Speed Index';
-  case 'fv':
-    return 'First Visual Change';
-  case 'vc':
-    return 'Visually Complete 100%';
-  case 'tti':
-    return 'Time to Interactive';
-  case 'vc85':
-    return 'Visually Complete 85%';
-  default:
-    throw new Error(`No matching message ID: ${messageType}`);
+    case 'NO_URL':
+      return 'No url entered.';
+    case 'LAUNCHING_CHROME':
+      return 'Launching Chrome';
+    case 'CLOSING_CHROME':
+      return '\nClosing Chrome';
+    case 'CRI_TIMEOUT_RELAUNCH':
+      return 'CRI_TIMEOUT error. Running Lighthouse one more time ...';
+    case 'CRI_TIMEOUT_REJECT':
+      return 'CRI_TIMEOUT error. Giving up running Lighthouse';
+    case 'MEDIAN_RUN':
+      return '                ☆  Median run  ☆';
+    case 'SAVED_TO_JSON':
+      return `Data was saved into file ${args[0]}`;
+    case 'NO_METRICS':
+      return 'No expectation metrics were found';
+    case 'NO_EXPECTATION_ERROR':
+      return `Metric ${args[0]} has to have warn and error values`;
+    case 'NO_EXPECTATIONS_FOUND':
+      return `expectation flag set but no expectations found on config file`;
+    case 'NO_SHEET_TYPE':
+      return `Sheet type ${args[0]} is not available.`;
+    case 'NO_GOOGLE_SHEET_OPTIONS':
+      return `Some of options for submitting data to Google Sheets are absent`;
+    case 'NO_MESSAGE_PREFIX_TYPE':
+      return `No matching message prefix: ${args[0]}`;
+    case 'METRIC_IS_UNAVAILABLE':
+      return `Sorry, ${args[0]} metric is unavailable`;
+    case 'ttfcp':
+      return 'First Contentful Paint';
+    case 'ttfmp':
+      return 'First Meaningful Paint';
+    case 'psi':
+      return 'Perceptual Speed Index';
+    case 'fv':
+      return 'First Visual Change';
+    case 'vc':
+      return 'Visually Complete 100%';
+    case 'tti':
+      return 'Time to Interactive';
+    case 'vc85':
+      return 'Visually Complete 85%';
+    case 'SUCCESS_RUN':
+      return `Run ${args[0][0] + 1} of ${args[0][1]} finished successfully`;
+    case 'FAILED_RUN':
+      return `Unable to complete run ${args[0][0] + 1} of ${args[0][1]} due to ${args[0][2]}`
```

Comment:
Here is quick (it's bad don't judge) code snippet demonstrating what was happening: 
https://repl.it/GNlK/0

 I fixed it in this [commit](https://github.com/paulirish/pwmetrics/pull/70/commits/f2bd2e1afa5f697fced9d5e5c3c934bc6de1b809). Doesn't seem to break anything, can anyone confirm @paulirish @denar90 ?